var/spack/repos/builtin/packages/laghos/package.py

```diff
@@ -53,6 +53,7 @@ def build_targets(self):
                        'share/mfem/config.mk'))
         targets.append('TEST_MK=%s' % join_path(spec['mfem'].prefix,
                        'share/mfem/test.mk'))
+        targets.append('CXX=%s' % spec['mpi'].mpicxx)
```

Comment:
@junghans, can you elaborate: why was this change necessary?

We are adding new versions of mfem and laghos in #15500 and this line is one of the reasons the laghos build breaks when mfem is built with cuda support.

Reply:
I don't recall exactly, but it had something to do with mpi on Fedora, just inject the option only for older versions and let's see what breaks.

Reply:
OK, thanks! I'll keep it for older versions. It should not be necessary but I guess there is some corner case I'm not seeing.